common/terms.html

```diff
@@ -144,6 +148,10 @@
     <dd>
       <p>An 'owning element' is any <abbr title="Document Object Model">DOM</abbr> ancestor of the <a>element</a>, or any element with an <pref>aria-owns</pref> attribute which references the ID of the   element. </p>
     </dd>
+    <dt><dfn data-lt="parent element|parent">Parent Element</dfn></dt>
+    <dd>
+      <p>A 'parent element' is a direct <abbr title="Document Object Model">DOM</abbr> parent of the <a>element</a>, an ancestor of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or any element with a child specified via <pref>aria-owns</pref>.</p>
```

Comment:
How about  "An 'owning parent'"? (To be more consistent with owned child, owned element, owning element)

As with 'owned child', do intermediate generics with aria-owns factor in?

(Also, one of these two could also refer to the other.)

Reply:
Updated, with a reference back to "owned child"